### PR TITLE
Bug 1790478 - Update repositories.yaml

### PR DESCRIPTION
**Pull request by evoluchico**
Add viu-politica app to repositories.yaml for probe scraper to validate and publish its metrics

### CODE REVIEW

File: repositories.yaml [hunk 1]
```diff
@@ -1083,3 +1083,25 @@ applications:
...
+    ping_files:
+      - source/telemetry/pings.yaml
+    dependencies:
+      - glean-js
+    moz_pipeline_metadata_defaults:
+      expiration_policy:
+        delete_after_days: 365
+    channels:
+      - viu.politica.prod
+      - viu.politica.dev
```
Comment (chutten) [inline repositories.yaml:1106]: If this is something you can control the channels of yourself, there's no need to specify channels here at all.

File: repositories.yaml [hunk 2]
```diff
@@ -1083,3 +1083,25 @@ applications:
       - v1_name: rally-attention-stream
         app_id: rally-attention-stream
     skip_documentation: true
+
+- app_name: viu_politica
+    canonical_app_name: Viu Politica
+    app_description: |
+      This is a "spin-off" of the Mozilla YouTube RegretsReporter extension. It is an opt-in browser extension where users can anonymously tag videos related to the 2022 Brazilian Elections.
```
Comment (chutten) [inline repositories.yaml:1090]: It's a multi-line field, so let's keep column widths down.

```suggestion
      This is a "spin-off" of the Mozilla YouTube RegretsReporter extension.
      It is an opt-in browser extension where users can anonymously tag videos related to the 2022 Brazilian Elections.
```

File: repositories.yaml [hunk 3]
```diff
@@ -1083,3 +1083,25 @@ applications:
       - v1_name: rally-attention-stream
         app_id: rally-attention-stream
     skip_documentation: true
+
+- app_name: viu_politica
```
Comment (chutten) [inline repositories.yaml:1087]: I believe this is missing some fields that'll allow this to properly run (v1_name and app_id, specifically). Did you happen to test this? Maybe with a command like `python -m probe_scraper.runner --glean --glean-repo viu_politica`?

**Review by chutten — CHANGES_REQUESTED** [review]

**Comment by Dexterp37** [comment]
Closed in favour of #496

**Pull request closed without merging**